catalog/MDCCatalog/AppDelegate.swift

```diff
@@ -58,7 +58,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     self.window?.rootViewController = navigationController
     self.window?.makeKeyAndVisible()
 
-    colorScheme = MDCBasicColorScheme(primaryColor: UIColor.init(white: 0.2, alpha: 1),
+    colorScheme = MDCBasicColorScheme(primaryColor: UIColor(red: 33 / 255.0,
+                                                            green: 33 / 255.0,
+                                                            blue: 33 / 255.0,
+                                                            alpha: 1),
```

Comment:
The rest of the parameters are given as UIColor white:alpha:
Consider UIColor(white:0.13 alpha:1)

Reply:
Done